src/dyn/dyn_control_params.h

```diff
@@ -456,6 +456,14 @@ class dyn_control_params{
   double fssh3_err_tol;
 
   
+  //=========== QTSH options ==========
+  /**
+    Whether to use QTSH

```

Comment:
add options possible and what they mean: 0 - not using QTSH; 1 - using QTSH  - explain to user what is actually changed when we use QTSH (a different convention for the force to use)